snapshot/lib/snapshot/reports_generator.rb

```diff
@@ -83,6 +86,8 @@ def xcode_9_and_above_device_name_mappings
         'iPad Pro (12.9-inch) (2nd generation)' => 'iPad Pro (12.9-inch) (2nd generation)',
         'iPad Pro (12.9-inch)' => 'iPad Pro (12.9-inch)',
         'Apple TV 1080p' => 'Apple TV',
```

Comment:
"Apple TV 1080p" as changed to "Apple TV", so think we need a mapping for that too?

Reply:
Indeed. But that's strange, when I tested on my Mac, I had an `Apple TV 1080p`. Right now, after a `reset_simulators`, I just have `Apple TV`. So, I think we should manage both. 

Since this PR is merged, I'll create a new one with this.

Reply:
Awesome, thanks @juli1quere 👍